### PR TITLE
optimize: extract the default values for some properties

### PR DESCRIPTION
**Pull request by tuwenlin**
extract the default values for some properties

### CODE REVIEW

**Review by slievrly — APPROVED** [review]
LGTM

**Comment by slievrly** [comment]
@tuwenlin In the future, I think we need to adjust the structure of default values to distinguish between client and server default values.

**Comment by tuwenlin** [comment]
> @tuwenlin In the future, I think we need to adjust the structure of default values to distinguish between client and server default values.

copy that

File: server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java [hunk 1]
```diff
@@ -132,13 +141,13 @@ public class DefaultCoordinator extends AbstractTCInboundHandler implements Tran
     private static final int BRANCH_ASYNC_POOL_SIZE = Runtime.getRuntime().availableProcessors();
 
     private static final Duration MAX_COMMIT_RETRY_TIMEOUT = ConfigurationFactory.getInstance().getDuration(
-            ConfigurationKeys.MAX_COMMIT_RETRY_TIMEOUT, DurationUtil.DEFAULT_DURATION, 100);
+            ConfigurationKeys.MAX_COMMIT_RETRY_TIMEOUT, DurationUtil.DEFAULT_DURATION, DEFAULT_MAX_COMMIT_RETRY_TIMEOUT);
 
     private static final Duration MAX_ROLLBACK_RETRY_TIMEOUT = ConfigurationFactory.getInstance().getDuration(
-            ConfigurationKeys.MAX_ROLLBACK_RETRY_TIMEOUT, DurationUtil.DEFAULT_DURATION, 100);
+            ConfigurationKeys.MAX_ROLLBACK_RETRY_TIMEOUT, DurationUtil.DEFAULT_DURATION, DEFAULT_MAX_ROLLBACK_RETRY_TIMEOUT);
```
Comment (wangliang181230) [inline server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java:147]: `DEFAULT_MAX_COMMIT_RETRY_TIMEOUT` and `DEFAULT_MAX_ROLLBACK_RETRY_TIMEOUT` is not `defaultValue` param, it's `timeoutMills`.

**Pull request merged**